cea/tests/test_calc_thermal_loads.config

```diff
@@ -1,7 +1,7 @@
 [test_calc_thermal_loads]
 value_columns = ["E_sys_kWh", "Qcdata_sys_kWh", "Qcre_sys_kWh", "Qcs_sys_kWh", "Qhs_sys_kWh", "Qww_sys_kWh", "Tcs_sys_re_C", "Ths_sys_re_C", "Tww_sys_re_C", "Tcs_sys_sup_C", "Ths_sys_sup_C", "Tww_sys_sup_C"]
-values = [333157.804, 0.0, 35057.826, 136873.46899999998, 164227.413, 24425.425, 0.0, 0.0, 26186.16, 0.0, 0.0, 171600.0]
+values = [332591.53400000004, 0.0, 35057.826, 136848.49699999997, 164146.24, 24425.275, 0.0, 0.0, 26186.16, 0.0, 0.0, 171600.0]
```

Comment:
why did it change? Qcs_sys? Qcpro should not contribute at all at this...

Reply:
It doesn't – but I also changed the process schedules now based on discussions in #1932 and #2053.